scripts/usb-policy.conf

```diff
@@ -5,9 +5,10 @@
 #  and each rule is (ALLOW | DENY) : ( match )*
 #  and each match is (class|subclass|prot|vid|pid|rel) = hex-number
 # Maximum hex value for class/subclass/prot is FF, and for vid/pid/rel is FFFF
```

Comment:
I think this is the right thing to do, but this commit should be part of a different PR and not of this one